Lesson10/src/by/home/service/Main.java

```diff
@@ -0,0 +1,25 @@
+package by.home.service;
+
+import by.home.model.CashDesk;
+import by.home.model.Customer;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) {
+
+        CashDesk cashDesk1 = new CashDesk(1);
+        CashDesk cashDesk2 = new CashDesk(2);
+        CashDesk cashDesk3 = new CashDesk(3);
+        List<CashDesk> cashDesks = new LinkedList<>();
+        cashDesks.add(cashDesk1);
+        cashDesks.add(cashDesk2);
+        cashDesks.add(cashDesk3);
+
+        for (int i = 1; i <= 4; i++) {
+            new Thread(new Customer(String.valueOf(i), cashDesks)).start();
```

Comment:
add Thread.sleep(100) after this line and your code should work. The thing is - you let all the customers in your shop in the same moment, so they all seeing the first cashdesk as an empty one) If you will wait a bit after each customer enters the shop - it should work